src/hooks/VoiceRecorder/index.tsx

```diff
@@ -34,11 +33,25 @@ const VoiceRecorderContext = createContext<VoiceRecorderContext>({
 export const VoiceRecorderProvider = (props: VoiceRecorderProps): React.ReactElement => {
   const { children } = props;
   const { config } = useSendbirdStateContext();
-  const { logger } = config;
+  const { logger, isVoiceMessageEnabled } = config;
   const [mediaRecorder, setMediaRecorder] = useState<MediaRecorder>(null);
   const [isRecordable, setIsRecordable] = useState<boolean>(false);
 
+  const [webAudioUtils, setWebAudioUtils] = useState(null);
+  useEffect(() => {
+    if (isVoiceMessageEnabled && !webAudioUtils) {
+      import('./WebAudioUtils').then((data) => {
+        setWebAudioUtils(data);
+      })
+    }
+  }, []);
```

Comment:
Maybe we can have a separate hook like 

```
const useLazyLoading({
  cond,
  module,
}) {
  const [data, setData] = useState(null);
  useEffect(() => {
    if (cond) {
      import(module).then((data) => {
        setData(data);
      })
    }
  }, []);
  
  return null;
}
```

and its usage would be like 
```
useLazyLoading({ cond: ..., module: ... });
```

Reply:
Yeah, that's a good idea, let me keep this topic to the next sprint